Makefile

```diff
@@ -61,7 +65,7 @@ endif
 ################################################################################
 .PHONY: test
 test:
-	go test ./... $(COVERAGE_OPTS) $(BUILDMODE)
+	CGO_ENABLED=1 go test -race -tags unit ./... $(COVERAGE_OPTS) $(BUILDMODE)
```

Comment:
Let’s have a separate command for this. This makes test run a lot slower. It is also using CGo which we don’t use to build the final binary.

Reply:
Done